terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -412,6 +410,7 @@ locals {
       "output_directory_germline": null,
       "output_directory_umccrise": null,
       "output_file_prefix_germline": null,
+      "enable_duplicate_marking": true,
```

Comment:
ditto 257, so all good, just noting.

Reply:
Have merged master back into this branch, so changes are now only relevant to this branch